docs/how-to/jupyter-notebook/list-status-notebooks.rst

```diff
@@ -0,0 +1,55 @@
+List Created Notebooks
+======================
+
+This guide explains how to use the `dss list` command to view the current status of all notebooks within the Data Science Stack (DSS) environment.
+
+Overview
+--------
+
+The `dss list` command provides a snapshot of all notebooks along with their current states. Understanding these states helps you manage and interact with your notebooks effectively.
+
+Using the `dss list` Command
+----------------------------
+
+To view the list and status of all notebooks, run the following command in your DSS environment:
+
+.. code-block:: bash
+
+    dss list
+
+This will display each notebook along with its current state and URL if applicable.
+
+Notebook States
+---------------
+
+Each notebook can be in one of the following states:
+
+- **Active**: The notebook is running and accessible. The URL under the "URL" column will be displayed, allowing you to access the notebook directly.
+
+- **Stopped**: The notebook is not running. You can start it using the `dss start` command.
+
+- **Stopping**: The notebook is in the process of stopping. It is advisable to wait until the process completes, transitioning to "Stopped".
+
+- **Starting**: The notebook is starting up. This state indicates that the notebook is initialising and will soon be active.
+
+- **Downloading**: The notebook is downloading necessary data or software. This is usually a transient state before it becomes "Active".
```

Comment:
Can we rephrase this to something like:
` The notebook is downloading the specified OCI Image. This is usually a .....`

To make it clear what's getting downloaded at this step?